swagger.json

```diff
@@ -3119,6 +3196,215 @@
+                            "type": "array",
+                            "description": "Power status of the reefer",
+                            "items": {
+                                "type": "object",
+                                "properties": {
+                                    "changedAtMs": {
+                                        "type": "integer",
+                                        "format": "int64",
+                                        "description": "Timestamp in Unix milliseconds since epoch.",
+                                        "example": 1453449599999
+                                    },
+                                    "status": {
+                                        "type": "string",
+                                        "description": "Power status of the reefer.",
+                                        "example": "Active (Continuous)"
+                                    }
+                                }
+                            }
+                        },
+                        "returnAirTemp": {
+                            "type": "array",
+                            "description": "Return air temperature of the reefer",
+                            "items": {
+                                "type": "object",
+                                "properties": {
+                                    "changedAtMs": {
+                                        "type": "integer",
+                                        "format": "int64",
+                                        "description": "Timestamp in Unix milliseconds since epoch.",
+                                        "example": 1453449599999
+                                    },
+                                    "tempInMilliC": {
+                                        "type": "integer",
+                                        "format": "int64",
+                                        "description": "Return air temperature in millidegree Celsius.",
+                                        "example": 31110
+                                    }
+                                }
+                            }
+                        },
+                        "setPoint": {
+                            "type": "array",
+                            "description": "Set point temperature of the reefer",
+                            "items": {
+                                "type": "object",
+                                "properties": {
+                                    "changedAtMs": {
+                                        "type": "integer",
+                                        "format": "int64",
+                                        "description": "Timestamp in Unix milliseconds since epoch.",
+                                        "example": 1453449599999
+                                    },
+                                    "setPointInMilliC": {
```

Comment:
just a small observation should be change this to temperatureInMilliC. nbd either would be fine.

Reply:
ok